conftest.py

```diff
@@ -195,7 +195,7 @@ def netcdf4_virtual_dataset(netcdf4_file):
     """Create a virtual dataset from a NetCDF4 file."""
     from virtualizarr import open_virtual_dataset
 
-    with open_virtual_dataset(netcdf4_file, indexes={}) as ds:
+    with open_virtual_dataset(netcdf4_file, loadable_variables=[]) as ds:
```

Comment:
Required otherwise we get inlined variables in the kerchunk file which we don't know how to read (#489)